warehouse/models/intermediate/gtfs_quality/guidelines_checks/int_gtfs_quality__data_license.sql

```diff
@@ -22,9 +22,11 @@ int_gtfs_quality__data_license AS (
             WHEN gtfs_datasets.type = "schedule" THEN {{ compliance_schedule() }}
             WHEN gtfs_datasets.type IN ("vehicle_positions", "trip_updates", "service_alerts") THEN {{ compliance_rt() }}
         END AS feature,
-        CASE manual_check__data_license
-            WHEN 'Yes' THEN {{ guidelines_pass_status() }}
-            WHEN 'No' THEN {{ guidelines_fail_status() }}
+        CASE
```

Comment:
@evansiroky could you confirm this logic checks out? Basically anything containing the  word "Permissive" passes, and containing "Restrictive" with the exact text "Missing" fails

Reply:
There are also some options of `CC-BY` and `ODL-BY` that should also have a passing status. Otherwise, it looks good.

Reply:
Thanks, I'm updating it to include those two values (in caps)